fix deny(missing_docs) error

leshow opened the pull request:
quick comment to fix build failing because of the #![deny(missing_docs)]

am I correct in assuming this will fix the error?


```
/home/leshow/.cargo/registry/src/github.com-1ecc6299db9ec823/leveldb-0.3.9/src/database/comparator.rs:23:11: 23:12 error: missing documentation for an associated type
/home/leshow/.cargo/registry/src/github.com-1ecc6299db9ec823/leveldb-0.3.9/src/database/comparator.rs:23      type K: Key;
                                                                                                                   ^
/home/leshow/.cargo/registry/src/github.com-1ecc6299db9ec823/leveldb-0.3.9/src/lib.rs:43:9: 43:21 note: lint level defined here
/home/leshow/.cargo/registry/src/github.com-1ecc6299db9ec823/leveldb-0.3.9/src/lib.rs:43 #![deny(missing_docs)]
```

skade commented:
I fixed that in parallel in the recent version, I just didn't push last night.

Thank you!

skade commented:
Thank you!

I fixed it during the porting work on `old_io`, though, so I'm closing this.

skade commented:
I added a CONTRIBUTORS file and added you - thanks for keeping track, keep patches and suggestions coming \o/.

leshow commented:
Thanks! I really appreciate it. We're using your lib for a small project
(just getting started now) so it's been great to learn Rust and  be part of
the community.

On 27 February 2015 at 06:53, Florian Gilcher <notifications@github.com>
wrote:

> I added a CONTRIBUTORS file and added you - thanks for keeping track, keep
> patches and suggestions coming \o/.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/skade/leveldb/pull/5#issuecomment-76383571>.
>



-- 
Evan Cameron
